src/main/kotlin/dsm/wemeet/global/socket/intercept/AuthorizeInterceptor.kt

```
@@ -0,0 +1,42 @@
+        attributes: MutableMap<String, Any>
+    ): Boolean {
+        val token = request.headers.getFirst("Authorization")
+            ?.removePrefix("Bearer ")?.trim() ?: run {
+            response.setStatusCode(HttpStatus.UNAUTHORIZED)
+            return false
+        }
+
+        return try {
+            attributes["email"] = jwtProvider.getJws(token).body.subject
+            true
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)